test_scripts/API/SendLocation/001_SendLocation_success.lua

```diff
@@ -0,0 +1,103 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_requirements/issues/3
+-- Use case: https://github.com/smartdevicelink/sdl_requirements/blob/master/detailed_docs/SetInteriorVehicleData.md
+-- Item: Use Case 1: Main Flow
+--
+-- Requirement summary:
+-- [SDL_RC] Set available control module settings SetInteriorVehicleData
+--
+-- Description:
+-- In case:
+-- 1) Application is registered with REMOTE_CONTROL appHMIType
+-- 2) and sends valid SetInteriorVehicleData RPC with valid parameters
+-- SDL must:
+-- 1) Transfer this request to HMI
+-- 2) Respond with <result_code> received from HMI
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common_send_location = require('test_scripts/API/SendLocation/common_send_location')
+
+--[[ Local Variables ]]
+local request_params = {
+    longitudeDegrees = 1.1,
+    latitudeDegrees = 1.1,
+    address = {
+        countryName = "countryName",
+        countryCode = "countryName",
+        postalCode = "postalCode",
+        administrativeArea = "administrativeArea",
+        subAdministrativeArea = "subAdministrativeArea",
+        locality = "locality",
+        subLocality = "subLocality",
+        thoroughfare = "thoroughfare",
+        subThoroughfare = "subThoroughfare"
+    },
+    timeStamp = {
```

Comment:
@Itileda I propose add all posible parameters in "happy path" test. 
That is why I ask you to add parameter "millisecond" to "timestamp" structure.

Reply:
@aderiabin , fixed in 9ae38a4203ca7367bfbafed1628d5cb1a9e9fe0d